atomic_reactor/util.py

```diff
@@ -861,6 +861,8 @@ def get_inspect_for_image(
         """
         all_man_digests = self.get_all_manifests(image)
 
+        config_digest: Optional[str] = None
```

Comment:
No need to set a value here, it can just be an annotation

Reply:
done